config/kubernetes-csi/org.yaml

```diff
@@ -551,6 +551,10 @@ teams:
     - msau42
     - saad-ali
     - xing-yang
+    - carlbraganza
+    - PrasadG193
+    - hairyhum
+    - rakshith-r
```

Comment:
I think being added to the external-snapshot-metadata-maintainers group should be sufficient for cutting releases. 
Since ihcsim is no longer active, can you remove him and replace him with another person (not adding so many people to admin group)?

Reply:
> I think being added to the external-snapshot-metadata-maintainers group should be sufficient for cutting releases. Since ihcsim is no longer active, can you remove him and replace him with another person (not adding so many people to admin group)?

okay,
I've added only @carlbraganza instead.
Please take a look.